CHANGELOG.md

```diff
@@ -1,7 +1,32 @@
 # Change Log
 
-Details changes in each release of EarlGrey. EarlGrey follows
-[semantic versioning](http://semver.org/).
+Details changes in each release of EarlGrey. EarlGrey follows [semantic versioning](http://semver.org/).
+
+## [1.8.0](https://github.com/google/EarlGrey/tree/1.7.2) (03/17/2017)
+
+```
+Baseline: [9dfb48b]
+   + [9dfb48b]: 1.8.0 Release
+```
+
+### New Features
+* Added a multi-finger swipe action.
+
+### Bug Fixes
+* Fixed issue with accessibility not working on iOS 9.1.
```

Comment:
accessibility spamming `Remote service does not respond to _accessibilityMachPort` message on iOS 9.1 device